src/services/chat/index.ts

```diff
@@ -74,13 +77,18 @@ const plugin: FastifyPluginAsyncTypebox<GraaspChatPluginOptions> = async (fastif
           params: { itemId },
           body,
         } = request;
-        const member = asDefined(user?.account);
-        return await db.transaction(async (manager) => {
-          const repositories = buildRepositories(manager);
-          const message = await chatService.postOne(member, repositories, itemId, body);
-          await actionChatService.postPostMessageAction(request, repositories, message);
-          return message;
-        });
+        const account = asDefined(user?.account);
+        if (isMember(account) || isGuest(account)) {
```

Comment:
Is there a way to factor this out (with the throw)?  Looks like we will need this often.